backends/alsa/alsa-backend.c

```diff
@@ -169,15 +184,99 @@ alsa_backend_finalize (GObject *object)
     G_OBJECT_CLASS (alsa_backend_parent_class)->finalize (object);
 }
 
+#ifdef HAVE_UDEV
+static gboolean udev_event_ok (struct udev_device *dev)
+{
+    const char *action = udev_device_get_action(dev);
```

Comment:
A space is missing before the bracket here and in a few other places, to be consistent with your and our  preferred code-style.

Reply:
Fixed.

Reply:
Thx.